docs/adrs/00005-license-export-analytics.md

```diff
@@ -0,0 +1,114 @@
+# 00005. Analysis of the license data structure.
+
+Date: 2025-01-23
+
+## Status
+
+DRAFT
+
+## Context
+### An SBOM typically contains three types of license information:
+
+#### 1 License information for the entire SBOM
```

Comment:
I think the main topic of this ADR is the license of individual components. So it's worth mentioning this as an exclusion later on. But drop it from the main section, if that's the case.